sessionctx/variable/tidb_vars.go

```diff
@@ -789,6 +789,10 @@ const (
 
 	// TiDBStoreBatchSize indicates the batch size of coprocessor in the same store.
 	TiDBStoreBatchSize = "tidb_store_batch_size"
+
+	// TiDBPessimisticTransactionAggressiveLocking controls whether aggressive locking for pessimistic transaction
+	// is enabled.
+	TiDBPessimisticTransactionAggressiveLocking = "tidb_pessimistic_transaction_aggressive_locking"
```

Comment:
Most TiDB system variables use `txn` to represent "transaction". I'm not sure if it's better to follow the convention...